src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md

```diff
@@ -1,18 +1,14 @@
-{% comment %}
-After the free trial details are finalized, add a link to free trial page to check for free trial eligibility.
-{% endcomment %}
-
-1. Create a [CockroachDB {{ site.data.products.cloud }}](https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}) account and log in.
-1. On the **Clusters** page, click **Create cluster**.
+1. [Create a CockroachDB {{ site.data.products.cloud }} account]({% link cockroachcloud/create-an-account.md %}). If this is your first CockroachDB Cloud organization, it will be credited with $400 in [free trial credits]({% link cockroachcloud/free-trial.md %}) to get you started.
```

Comment:
Noting that this content must somehow be kept up to date if the actual amount of credits changes.

Reply:
Ack. We will have to search for other mentions anyway.